docs/configuration.md

```diff
@@ -0,0 +1,71 @@
+# Configuring Spectral
+
+## Example config file (spectral.yaml)
+
+```yaml
+rules:
+    - rule-name:
+        given: $..parameters[*]
+        then:
+            field: description
+            function: truthy        
+```
+
+## Rules
+
+Rules are highly configurable. There are only few required parameters but the optional ones gives powerful flexibility.
```

Comment:
If the required parameters are `few` I'd call them out here.

Reply:
I will prepare a table describing Rule (similar to what you can find in OAS documentation)

Reply:
that's done